content/source/docs/enterprise-beta/sentinel/enforce.html.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: enterprise2
+page_title: "Sentinel - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-sentinel-enforce"
+---
+
+# Enforce and Override Policies
+
+Once a policy is added to an organization it is enforced on all Workspace Runs.
```

Comment:
maybe just runs?

Reply:
Might also want to explicitly add that you must upload the policy before it can be enforced. Creation alone isn't enough.